lib/IRGen/IRGenDebugInfo.h

```diff
@@ -38,7 +38,12 @@ class IRBuilder;
 class IRGenFunction;
 class IRGenModule;
 
-enum IndirectionKind : bool { DirectValue = false, IndirectValue = true };
+enum IndirectionKind {
+  DirectValue,
+  IndirectValue,
+  InCoroContext,
+  InCoroContextIndirect
```

Comment:
Total nitpick: wdyt of replacing {'InCoroContext', 'InCoroContextIndirect'} with {'CoroDirectValue' and 'CoroIndirectValue'}? Naming might feel a bit more similar that way. Another option might be an enum class, then you could spell 'IndirectionKind::Direct', 'IndirectionKind::CoroDirect' etc.